.github/workflows/core-hadoop3-ci.yml

```diff
@@ -37,9 +37,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        jdk: [ '8', '11' ]
-        spark: [ '3.3', '3.5' ]
-    name: Build Amoro with JDK ${{ matrix.jdk }} Spark-${{ matrix.spark }}
+        include:
```

Comment:
Can you remove the change of ci? we can update this in other PR

Reply:
done